nested.py

```diff
@@ -1,22 +1,66 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 """
-Module docstring: One line description of what your program does.
+checks whether expressions are properly nested. If the expression is not
+properly nested program should determine the position of the offending bracket.
 """
-__author__ = "???"
+__author__ = "Avelyno Koumaka"
 
 import sys
 
+open_list = ["(*", "(", "[", "{", "<"]
+close_list = ["*)", ")", "]", "}", ">"]
+
 
 def is_nested(line):
     """Validate a single input line for correct nesting"""
-    pass
+    stack = []
+    count = 1
+
+    def validate_open_list(token, stack):
```

Comment:
Any custom functions that you create are best defined in the global scope. (i.e., `def validate_open_list` should be placed outside of the `is_nested` function -- the same goes for `def validate_close_list`).